devito/types/dimension.py

```diff
@@ -1193,6 +1261,59 @@ def _symbolic_thickness(cls, name):
                 Scalar(name="%s_rtkn" % name, dtype=np.int32, nonnegative=True))
 
 
+class StencilDimension(BasicDimension):
+
+    """
+    Dimension symbol representing the points of a stencil.
+
+    Parameters
+    ----------
+    name : str
+        Name of the dimension.
+    _min : expr-like
+        The minimum point of the stencil.
+    _max : expr-like
+        The maximum point of the stencil.
+    """
+
+    def __init_finalize__(self, name, _min, _max):
+        self._spacing = sympy.S.One
```

Comment:
would make this oneinput as well for non unitary steps

Reply:
Right. Done and added test